application/modules/quotes/views/partial_itemlist_responsive.php

```diff
@@ -0,0 +1,318 @@
+            </a>
+            <a href="javascript:void(0);" class="btn_add_product btn btn-sm btn-default">
+              <i class="fa fa-database"></i><?php _trans('add_product'); ?>
+            </a>
+        </div>
+    </div>
+
+		<div class="col-xs-12 visible-xs visible-sm"><br></div>
+
+    <div class="col-xs-12 col-md-6 col-md-offset-2 col-lg-4 col-lg-offset-4">
+        <table class="table table-bordered text-right">
+            <tr>
+                <td style="width: 40%;"><?php _trans('subtotal'); ?></td>
+                <td style="width: 60%;" class="amount"><?php echo format_currency($quote->quote_item_subtotal); ?></td>
+            </tr>
+            <tr>
+                <td><?php _trans('item_tax'); ?></td>
+                <td class="amount"><?php echo format_currency($quote->quote_item_tax_total); ?></td>
+            </tr>
+            <tr>
+                <td><?php _trans('quote_tax'); ?></td>
+                <td>
+                    <?php if ($quote_tax_rates) {
+                        foreach ($quote_tax_rates as $index => $quote_tax_rate) { ?>
+                            <form method="POST" class="form-inline"
+                                  action="<?php echo site_url('quotes/delete_quote_tax/' . $quote->quote_id . '/' . $quote_tax_rate->quote_tax_rate_id) ?>">
+                                <?php _csrf_field(); ?>
+                                <span class="amount">
+                                    <?php echo format_currency($quote_tax_rate->quote_tax_rate_amount); ?>
+                                </span>
+                                <span class="text-muted">
+                                    <?php echo htmlsc($quote_tax_rate->quote_tax_rate_name) . ' ' . format_amount($quote_tax_rate->quote_tax_rate_percent) ?>
+                                </span>
+                                <button type="submit" class="btn btn-xs btn-link"
+                                        onclick="return confirm('<?php _trans('delete_tax_warning'); ?>');">
+                                    <i class="fa fa-trash-o"></i>
+                                </button>
+                            </form>
+                        <?php }
+                    } else {
+                        echo format_currency('0');
+                    } ?>
+                </td>
+            </tr>
+            <tr>
+                <td class="td-vert-middle"><?php _trans('discount'); ?></td>
+                <td class="clearfix">
+                    <div class="discount-field">
+                        <div class="input-group input-group-sm">
+                            <input id="quote_discount_amount" name="quote_discount_amount"
+                                   class="discount-option form-control input-sm amount"
+                                   value="<?php echo format_amount($quote->quote_discount_amount != 0 ? $quote->quote_discount_amount : ''); ?>">

```

Comment:
Added label to the inpute, afterwards hiding it with css, because in this case it's not intended to show the label.
But most of the mentioned errors are already inside the original code. I see two solutions: someone takes a closer look at the original (mainly moved to `partial_itemlist_table.php`), or we deprecate the old view.

Reply:
Yes you dont want the label cause you got the currency symbol to the right of the input.
But if it is intended to make the project accessible to everyone we would need a label to be read.
Since this is an artifact from old code we can either fix it there as well (and probably a lot of other spots) or keep it as is (nobody complained of screen reading issues or am i wrong?) and just keep this in mind for v2.0

Correct me if my ideas and thought are not at all comparable to the current project standards since I try to get into all of this in the time of the last week.
(And my English is horrible sometimes :P)

Reply:
So now it's there, hidden with css and added with translation, so can be evaluated by screen reader. I think, this is good to go now.